SDKTest/NIDPerformanceTests.swift

```diff
@@ -21,8 +21,7 @@ final class NIDPerformanceTests: XCTestCase {
 
     override func setUp() {
         NeuroID.networkService = NIDNetworkServiceTestImpl()
-
-        NeuroID._isSDKStarted = true
+        NeuroID.start()
```

Comment:
Do we want to call start instead of just setting the variable to true that allows data collection? With the throttling work if we want to call start we need to mock the config service to immediately return a result